src/main/kotlin/no/nav/familie/ef/sak/iverksett/oppgaveforbarn/ForberedOppgaverForBarnTask.kt

```diff
@@ -0,0 +1,52 @@
+package no.nav.familie.ef.sak.iverksett.oppgaveforbarn
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.familie.ef.sak.infrastruktur.featuretoggle.FeatureToggleService
+import no.nav.familie.kontrakter.felles.objectMapper
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import no.nav.familie.prosessering.domene.TaskRepository
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+@Service
+@TaskStepBeskrivelse(
+        taskStepType = ForberedOppgaverForBarnTask.TYPE,
+        beskrivelse = "Oppretter oppgave for barn som fyller 1/2 eller 1 år",
+
+        )
+class ForberedOppgaverForBarnTask(val taskRepository: TaskRepository,
```

Comment:
Hvordan blir denne initiert? (opprettet for første gang)

Reply:
Jeg la til en : .../nav/familie/ef/sak/iverksett/oppgaveforbarn/InitForberedOppgaverForBarnTaskController.kt . Er det noe sånt du tenker på for å få startet oppgaven ?